libs/ui/src/lib/snackbar/snackbar.tsx

```diff
@@ -0,0 +1,33 @@
+import {Alert, AlertColor, Snackbar, SnackbarProps} from '@mui/material'
+import {createContext, useState} from 'react'
+
+export const SnackbarContext = createContext<{
+  setSnack: React.Dispatch<React.SetStateAction<Snack | undefined>>
+}>({
+  setSnack: () => {
+    return
+  }
+})
+
+export interface Snack {
+  message: string
+  severity?: AlertColor
+}
+
+export function SnackbarWithContextProvider({children, ...props}: SnackbarProps) {
```

Comment:
```suggestion
export function SnackbarWithContextProvider({children, autoHideDuration = 10000, ...props}: SnackbarProps) {
```

Then you don't need DEFAULT_AUTO_HIDE_DURATION